src/components/skeleton/article.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { makeStyles } from '../../utils/make-styles'
+
+const articleStyles = (skeleton, customSkeleton) => { 
+  return makeStyles({
+    title: {
+      width: '50%',
+      height: '1.25rem',
+      marginBottom: '0.938rem',
+    },
+    text: {
+      width: '100%',
+      height: '0.75rem',
```

Comment:
lets try to use as much of existing classes from tailwind if it already exits in tailwind config lets use them. Only do it in makestyle way for complicated cases or child inner container. see existing components for examples. https://tailwindcss.com/docs/width classes from tailwind also you can take look in tailwindConfig.js